internal/image/validate.go

```diff
@@ -36,14 +37,24 @@ func ValidateImage(ctx context.Context, imageRef, policyConfiguration, publicKey
 		return nil, err
 	}
 
+	if err = a.ValidateImageAccess(ctx); err != nil {
+		log.Debugf("Image access check failed. Error: %s", err.Error())
+		out.SetImageAccessibleCheck(false, fmt.Sprintf("image ref not accessible. %s", err))
+		out.SetImageSignatureCheck(false, "")
```

Comment:
I don't think this is needed, right? It should default to `false`.

Reply:
If you're referring to the `false` argument provided to `out.SetImageSignatureCheck()`, it is required. While the default for a `boolean` is `false`, you can't have default parameter values in method signatures, so both parameters "passed" and "message" must be provided.

Reply:
Oh, that's not what I meant. I meant omitting the call to `out.SetImageSignatureCheck` altogether. The underlying default state for the `ImageSignatureCheck` should be false by default.

Reply:
Turns out that skipping any of the following results in a nil pointer dereference:
```
out.SetImageSignatureCheck(false, "")
out.SetAttestationSignatureCheck(false, "")
out.SetPolicyCheck(nil)
```